lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/grok/Garbage.java

```diff
@@ -84,22 +83,22 @@ public int rename(Map<String, Object> map)
     {
         int item = 0;
 
-        if (map == null) {
+        if (map == null || map.isEmpty() || toRename.isEmpty()) {
```

Comment:
The check below for `map.isEmpty() || toRename.isEmpty()` is now redundant and should be removed.